lib/collection/src/shards/replica_set/shard_transfer.rs

```diff
@@ -20,7 +20,11 @@ impl ShardReplicaSet {
     /// # Cancel safety
     ///
     /// This method is cancel safe.
-    pub async fn proxify_local(&self, remote_shard: RemoteShard) -> CollectionResult<()> {
+    pub async fn proxify_local(
+        &self,
+        remote_shard: RemoteShard,
+        hash_ring: Option<HashRing>,
```

Comment:
let's mention, that it is a new hashring

Reply:
Renamed to `resharding_hash_ring`.